capstone/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst

```diff
@@ -1 +1,10 @@
-/home/anikabagga/capstone-project/capstone/src/main/java/com/google/sps/servlets/DataServlet.java
+/home/anikabagga/step100-2020/capstone/src/main/java/com/google/sps/Objects/User.java
```

Comment:
This file appears like it shouldn't be uploaded, since it directly references /home/anikabagga/...

Reply:
removed target folder